matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -754,6 +766,7 @@ internal class LocalEchoEventFactory @Inject constructor(
             MessageType.MSGTYPE_POLL_START -> {
                 return TextContent((content as? MessagePollContent)?.getBestPollCreationInfo()?.question?.getBestQuestion() ?: "")
             }
+            MessageType.MSGTYPE_POLL_END -> return TextContent("Ended poll")
```

Comment:
Maybe we should use lower case for the start like `ended poll` to follow convention of other contents?

Reply:
Design for this suggests to use uppercase but I wouldn't mind to make it lowercase.